timer.js

```diff
@@ -113,13 +126,16 @@ document.addEventListener('DOMContentLoaded', function() {
 
   var activeTimerList = new TimerList(document.querySelector('ul.active-timers'));
 
-  var finishedTimerList = new TimerList(document.querySelector('ul.completed-timers'), function(timer) {
-    return 'Timer ' + timer['id'] + ' expired ' + timeDiff(timer['expires'])+ ' seconds ago';
+  var finishedTimerList = new TimerList(document.querySelector('ul.completed-timers'), {
+    countdownFormatter: function(timer) {
+      return 'Timer ' + timer['id'] + ' expired ' + timeDiff(timer['expires'])+ ' seconds ago';
+    }
   });
 
   var masterTimer = new MasterTimer({
     onTimerComplete: function(timer) {
-      alert('timer ' + timer['id'] + ' completed');
+      var alarm = new Audio('alarm.mp3');
```

Comment:
Cool! The alerts were annoying, but good enough for my actual needs. This is better.